### PR TITLE
Gatsby example

### PR DESCRIPTION
**Pull request by pusztaienike**

### CODE REVIEW

File: examples/sn-gatsby-example/gatsby-config.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      resolve: 'gatsby-transformer-remark',
+      options: {
+        plugins: [
+          {
+            resolve: 'gatsby-remark-responsive-iframe',
+            options: {
+              wrapperStyle: 'margin-bottom: 1rem',
+            },
+          },
+          'gatsby-remark-prismjs',
```
Comment (taki9) [inline examples/sn-gatsby-example/gatsby-config.js:31]: does syntax highlighting work?

File: examples/sn-gatsby-example/src/layouts/index.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      `}
+      render={(data: StaticQueryProps) => (
+        <>
+          <CssBaseline />
+          <LayoutRoot>
+            <Helmet
+              title={data.site.siteMetadata.title}
+              meta={[
+                { name: 'description', content: data.site.siteMetadata.description },
+                { name: 'keywords', content: data.site.siteMetadata.keywords },
```
Comment (taki9) [inline examples/sn-gatsby-example/src/layouts/index.tsx:51]: it is not requested in the query

File: examples/sn-gatsby-example/src/components/blog-card.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    },
+    blogCardTitle: {
+      margin: '1rem 0',
+      fontSize: '20px',
+      fontWeight: 700,
+    },
+    blogCardDescription: {
+      marginBottom: '1rem',
+    },
+  })
```
Comment (taki9) [inline examples/sn-gatsby-example/src/components/blog-card.tsx:33]: use root, image, content/wrapper, title, description keys

File: examples/sn-gatsby-example/gatsby-config.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    'gatsby-transformer-json',
+    {
+      resolve: 'gatsby-plugin-canonical-urls',
+      options: {
+        siteUrl: 'https://gatsby-starter-typescript-plus.netlify.com',
+      },
+    },
+    'gatsby-plugin-typescript',
+    'gatsby-plugin-sharp',
+    'gatsby-transformer-sharp',
```
Comment (taki9) [inline examples/sn-gatsby-example/gatsby-config.js:54]: https://www.gatsbyjs.com/docs/how-to/images-and-media/using-gatsby-plugin-image/

right now you don't use it in the project

File: examples/sn-gatsby-example/gatsby-config.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+          'gatsby-remark-copy-linked-files',
+          'gatsby-remark-smartypants',
+          {
+            resolve: 'gatsby-remark-images',
+            options: {
+              maxWidth: 1140,
+              quality: 90,
+              linkImagesToOriginal: false,
+            },
+          },
```
Comment (taki9) [inline examples/sn-gatsby-example/gatsby-config.js:41]: is it working as expected? (

**Review by taki9 — COMMENTED** [review]
the project folder should be 'sn-blog-gatsby'

File: examples/sn-gatsby-example/gatsby-config.js (hunk 1 above)
Comment (pusztaienike) [reply examples/sn-gatsby-example/gatsby-config.js:31]: I had to add a theme (to gatsby-browser.js), now it is working require('prismjs/themes/prism-solarizedlight.css')

File: examples/sn-gatsby-example/gatsby-config.js (hunk 4 above)
Comment (pusztaienike) [reply examples/sn-gatsby-example/gatsby-config.js:54]: 'gatsby-plugin-sharp', is needed the other one has been removed
image

File: examples/sn-gatsby-example/gatsby-config.js (hunk 5 above)
Comment (pusztaienike) [reply examples/sn-gatsby-example/gatsby-config.js:41]: no. (removed)

File: examples/sn-blog-gatsby/package.json [hunk 6]
```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@sensenet/gatsby_example",
```
Comment (taki9) [inline examples/sn-blog-gatsby/package.json:2]: @example/blog-gatsby

**Review by taki9 — CHANGES_REQUESTED** [review]
Remove duplicated sn-universal-header package.

**Review by taki9 — APPROVED** [review]

**Review by herflis — APPROVED** [review]

**Pull request merged**